addons/l10n_cn/data/account.group.template.csv

```diff
@@ -0,0 +1 @@
+id,code_prefix_start,code_prefix_end,name,chart_template_id/id
```

Comment:
This file is unused. Also please make sure that all files end with a newline :)

Reply:
have add group templates